Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 
 Bugfixes:
  * Commandline Interface: Disallow the following options outside of the compiler mode: ``--via-ir``,``--metadata-literal``, ``--metadata-hash``, ``--model-checker-show-unproved``, ``--model-checker-div-mod-no-slacks``, ``--model-checker-engine``, ``--model-checker-invariants``, ``--model-checker-solvers``, ``--model-checker-timeout``, ``--model-checker-contracts``, ``--model-checker-targets``.
+ * TypeChecker: Fix null dereference in `abi.encodeCall` type checking of free function.
```

Comment:
```suggestion
 * Type Checker: Fix null dereference in `abi.encodeCall` type checking of free function.
```
(I know it is written like that above, but this is how it should be :) )

Reply:
Fixed :)